src/App.jsx

```diff
@@ -1,10 +1,25 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+
+// Components.
+import Main from './components/Main/Main';
+
 
 class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      fallingPieceX: 10,
+      fallingPieceY: 50
+    };
+  }
+
   render() {
     return (
-      <h1>Hello React :)</h1>
+      <Main {...this.state} />
```

Comment:
Al componente Main le paso el state te la applicación completo, a main le va a llegar cada uno de los fields del state en this.props.nombreDelField. Los tres puntitos adelante (...) no es algo de react es algo que se llama Destructuring que para React sirve mucho. Se agregó a js en ES6.